### DISCUSSION TITLE
Passband for Luminance filter?

### DISCUSSION THREAD

**Discussion by astro-sharp** [phoebe-project/phoebe2#818] [Questions]
Hi all,

In my research on PCEB binary systems, I regularly use a Luminance filter (i.e. an IR blocking filter) to get as much light as possible as we are interested in getting exposures captured as quickly as possible for the best temporal resolution on the light curve.

When I add my data with the `b.add_dataset()` method, is there a passband I can specify that would suit this filter? I can't immediately see one.

Many thanks
Ian.

**Comment by astro-sharp** [comment]
OK, so it must be to do with adding a custom passband? I'm trying to understand how to do this...
Any help much appreciated!
Ian.

**Comment by aprsa** [comment]
@astro-sharp, indeed we currently do not have such a passband supported. You can either try to create your custom passband (here's a [tutorial](http://phoebe-project.org/docs/latest/tutorials/passbands) to follow) or you can send me the transmission file (2-column format, wavelength and transmission) and I can create one for you, and add it to the passband repository.

**Reply by astro-sharp** [reply]
Many thanks for the reply. I have created and attached a transmission file - it's very simple as the passband is very close to a square-wave. (I based it on the Astronomik Type-II L filter which is very typical).

Is this OK for you to add to the passband repository? I'm getting errors when I try to run the tutorial on this line:
`pb.impute_atmosphere_grid(pb._phoenix_energy_grid)
`
![image](https://github.com/phoebe-project/phoebe2/assets/55244395/4dbc61b1-2577-42cd-a632-983b8d479de8)
[lum_passband.zip](https://github.com/phoebe-project/phoebe2/files/13988584/lum_passband.zip)
